src/pages/auth/Login.js

```diff
@@ -76,12 +113,16 @@ const Login = () => {
         </ul>
         <button
           className="button primary-button-pink my-2 text-xs"
-          onClick={async(e) => {
-        
-          }}
+          onClick={(e) => loginHandler(e, formData.email, formData.password)}
         >
           Login
         </button>
+        <button
+          className="button primary-button-pink mb-2 text-xs"
+          onClick={(e) => loginHandler(e, "priya@gmail.com", "priya123")}
```

Comment:
You have accessed to `formData` in  `loginHandler` no need to pass it again as parameter

Reply:
As I want to implement login as guest functionality where I will have to pass my own email and password rather than input fields email and password .